### PR TITLE
docs: add description about histograms graph

### PR DESCRIPTION
**Pull request by emori-ctc**
## Pre-review checklist for the PR author

- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [ ] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: docs/visualization/latency/index.md [hunk 1]
```diff
@@ -42,9 +44,23 @@ plot.show()
...
+### Histogram graph
+plot = Plot.create_latency_histogram_plot(app.callbacks)
+plot.show()
+
+# ---Output in jupyter-notebook as below---
+```
+
+![callback_latency_histogram](../../imgs/callback_latency_histogram.png)
+
+The horizontal axis means latency, labeled as `Latency [ms]`. The vertical axis is the Probability of delay, labeled as `probability`.
```
Comment (rokamu623) [inline docs/visualization/latency/index.md:59]: I would like to arrange it in the form of follow:

```The vertical axis is the probability of execution [hogehoge], labeled `probability`.```

**Review by rokamu623 — CHANGES_REQUESTED** [review]
The following modification is proposed:

**Review by bopeng-saitama — CHANGES_REQUESTED** [review]

**Review by rokamu623 — APPROVED** [review]
LGTM!

**Review by bopeng-saitama — APPROVED** [review]
LGTM

**Review by nabetetsu — APPROVED** [review]
LGTM :+1:

**Pull request merged**